### PR TITLE
Ash/country redirection

### PR DESCRIPTION
**Pull request by Ashhadsmallaxe**
Added language detection by users country.

### CODE REVIEW

File: app/js/script.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+$(function(){
+	var url = 'http://freegeoip.net/json/';
+      var xhr = new XMLHttpRequest();
+      xhr.open('GET', url);
+      console.log(xhr);
```
Comment (nishan-mills) [inline app/js/script.js]: @Ashhadsmallaxe can we remove these debug statements?

File: app/js/script.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      // Response handlers.
+      xhr.onload = function() {
+        var data = JSON.parse(xhr.responseText);
+        showNoticeByCountry(data.country_name);
+      };
+      xhr.onerror = function() {
+        alert('Woops, there was an error making the request.');
+      };
+      xhr.send();
+
```
Comment (nishan-mills) [inline app/js/script.js]: @Ashhadsmallaxe can we remove redundant blank lines?

File: app/js/script.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+});
+
+
+function showNoticeByCountry(country){
+	console.log('visitorcountry', country);
```
Comment (nishan-mills) [inline app/js/script.js]: @Ashhadsmallaxe can we remove debug statements?

File: app/js/script.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      xhr.send();
+
+});
+
+
+function showNoticeByCountry(country){
+	console.log('visitorcountry', country); 
+    $('.content').removeClass('active');
+	switch (country) {
+    case "Netherlands":
```
Comment (nishan-mills) [inline app/js/script.js]: @Ashhadsmallaxe when we are doing string comparisons, lets try and do them in lowercase (or uppercase); the input values, make sure to sanitize them by trim and lowercase in turn.

File: app/js/script.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    case "France":
+        $('.content.fr').addClass('active');
+        break;
+    case "Romania":
+        $('.content.ro').addClass('active');
+        break;
+    case "Germany":
+        $('.content.ge').addClass('active');
+        break;
+    case "Germany":
```
Comment (nishan-mills) [inline app/js/script.js]: @Ashhadsmallaxe we seem to have two cases for Germany?

File: app/js/script.js [hunk 6]
```diff
@@ -22,29 +21,23 @@ $(function(){
...
+
+    var countryMap = {
+    	"netherlands" : "de",
+    	"france" : "fr",
+    	"romania" : "ro",
+    	"germany" : 'ge',
+    	"bulgaria": 'bg',
+    }
+
+    if(countryMap.hasOwnProperty(country)){
```
Comment (nishan-mills) [inline app/js/script.js]: @Ashhadsmallaxe we could add another kv pair to the map called default and then set that to en, if there is no country then we pull up the default value.

**Comment by Ashhadsmallaxe** [comment]
@nishan-mills hope this is what you meant

**Pull request merged**